nativelib/src/main/resources/scala-native/compile_flags.txt

```diff
@@ -0,0 +1,33 @@
+# GC setup
```

Comment:
This file should not be included

Reply:
Have you seen what that can do with `clangd`? It could really help developers. We could always ignore it and have some docs so Scala Native developers could use it.

Reply:
ok, removed it and added a .gitignore